### PR TITLE
Merge 2.9

### PR DESCRIPTION
**Pull request by wallyworld**
Merge 2.9

#14568 equinix: set default deny iptables policy
#14858 [JUJU-2101] Fixed teardown and increased timeout for wait_for (test-deploy-magma)
#14865 Fix trace log message
#14869 [JUJU-2157] Fixed teardown issue in magma test

Conflicts
```
#       go.mod
#       go.sum
#       provider/equinix/environ.go
#       provider/equinix/environ_test.go
#
```

[JUJU-2101]: https://warthogs.atlassian.net/browse/JUJU-2101?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[JUJU-2157]: https://warthogs.atlassian.net/browse/JUJU-2157?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by anvial — APPROVED** [review]
lg2m

File: provider/equinix/environ.go [hunk 1]
```diff
@@ -302,20 +264,52 @@ func (e *environ) StartInstance(ctx context.ProviderCallContext, args environs.S
...
+	acceptInputPort := "iptables -A INPUT -p tcp --dport %d -j ACCEPT"
+	iptablesDefault := []string{
+		"iptables -A INPUT -m conntrack --ctstate INVALID -j DROP",
+		"iptables -A INPUT -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT",
+		"iptables -A OUTPUT -m conntrack --ctstate ESTABLISHED -j ACCEPT",
+		"iptables -A INPUT -p icmp -j ACCEPT",
+		"iptables -A INPUT -i lo -j ACCEPT",
+		"iptables -A OUTPUT -o lo -j ACCEPT",
+		"iptables -P INPUT ! -i lo -s 127.0.0.0/8 -j REJECT",
+		fmt.Sprintf("iptables -A OUTPUT -p tcp --sport %d -m conntrack --ctstate ESTABLISHED -j ACCEPT", ssh.SSHPort),
```
Comment (tlm) [inline provider/equinix/environ.go:287]: I think we should fix this PR up before bringing it across. I don't like that a provider has a double reference into the cmd code.

We should just make this a const in the package.

Comment (wallyworld) [reply provider/equinix/environ.go]: i've extracted a const for the iptables commands

**Pull request merged**